package.json

```diff
@@ -154,6 +154,6 @@
     "webpack-hot-middleware": "^2.24.3"
   },
   "workspaces": {
-    "packages": ["packages/*", "scripts"]
+    "packages": ["packages/*", "scripts", "digest", "perf-test"]
```

Comment:
can we place these under packages/ as well so we don't really have anything other than scripts at the root?

Reply:
Per my PR description above I couldn't put them in `packages` because it causes existing build tasks to fail, even when they are added to the exclusion list. (Unless there is another way to exclude that I could not find.)